io.github.waylyrics.Waylyrics.json

```diff
@@ -0,0 +1,60 @@
+{
+    "app-id": "io.github.waylyrics.Waylyrics",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "46",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+    "command": "waylyrics",
+    "finish-args": [
+        "--socket=wayland",
+        "--socket=fallback-x11",
+        "--device=dri",
+        "--share=ipc",
+        "--share=network",
+        "--talk-name=org.mpris.MediaPlayer2.*",
+        "--own-name=org.kde.*",
```

Comment:
why do you need to own this?

Reply:
this is the build failure.

Reply:
> why do you need to own this?

org.kde.* is actually matching `org.kde.NotificationItem-$PID-$ITEM`, and passing `org.kde.NotificationItem-*` will not work

Reply:
> why do you need to own this?

According to [desktop integration](https://docs.flatpak.org/en/latest/desktop-integration.html#statusnotifier), we cannot pass `org.kde.StatusNotifierItem-*` to own an tray icon dbus instance

Reply:
also this is a gtk app, isn't it?

Reply:
> also this is a gtk app, isn't it?

sure

Reply:
@hfiguiere I have submitted a pull request for adding the exception https://github.com/flathub-infra/flatpak-builder-lint/pull/385

Reply:
When I said it would never work and that was the reason of the build, I was saying that it should be removed. It will never work there is no reason to access the whole dbus for that.

Reply:
> When I said it would never work and that was the reason of the build, I was saying that it should be removed. It will never work there is no reason to access the whole dbus for that.

Oki. So I will suggest Integral to disable tray-icon feature for flathub build

Reply:
or to use the more modern API as mentionned in the documented you pointed to.

Reply:
in any case it should fail gracefuly

Reply:
Fixed.